src/main/kotlin/no/nav/familie/ef/iverksett/felles/FamilieIntegrasjonerClient.kt

```diff
@@ -40,8 +43,14 @@ class FamilieIntegrasjonerClient(
         return response.getDataOrThrow()["aktørId"].toString()
     }
 
-    companion object {
+    fun hentNavEnhet(personident: String): Enhet? {
```

Comment:
Trenger vi å hente navenheten eller finnes den allerede i iverksettingsobjektet? Tror det er det som er `tilhørendeEnhet`

Reply:
Dobbeltsjekket hva som blir hentet ut fra sak, og det er en liten detalj som gjør at det er annen enhet enn det som blir sendt med IverksettDto. Slik jeg har forstått det, er bruker koblet til flere NAV-kontor, den ene er saksbehandlende kontor og andre er lokalkontor. I dette tilfellet ønsket lokalkontoret. I kall mot integrasjon sendes det med tema OPP (oppfølging) i stedet for ENF i request, og det resulterer i annen nav-enhet enn det vi har i dto.
Mulig at det kan være greit å ha begge deler i dto, og at kanskje dette burde vært løst i ef-sak i stedet?